website/docs/modules/Info2.md

```diff
@@ -0,0 +1,32 @@
+---
+id: settings
+title: Settings
+sidebar_position: 1
+---
+
+## Module Description
+
+This module provides the abilitity to create and manage multiple rich-text pages for user information pages
+
+## Configuration options
+
+TBD
```

Comment:
Is this still work in progress?

Reply:
No I considered it really to be done in the future when also the other issues of the Info2 module are to be touched or we tackle generic module documentation.

I'm fine to remove the block if that is better?

Reply:
Yeah, lets remove it.

Reply:
I've decided to spend the few minutes to document it instead and noticed that option "extended toolbar" is not being used anywhere in the code, thus removed the option and documented the rest